src/containers/BrowseMoviesContainer.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import BrowsePage from "../components/BrowsePage";
-import { getMoviesFromType, getGenreMovies, getMovieGenres } from "../api/APIUtils";
+import { getMoviesFromType, getGenreMovies, getMovieGenres, getMoviesFromYear } from "../api/APIUtils";
+import createDebouncedFunc from "../utils/createDebouncedFunc";
 
 class BrowseMoviesContainer extends Component {
-  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: false };
+  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: "", searchWords: "" };
```

Comment:
Please put the state properties on their own line, this is really hard to read when there are so many values if they are on the same line